src/main/java/com/minecolonies/util/StructureWrapper.java

```diff
@@ -148,69 +131,49 @@ private static InputStream getStream(@NotNull ResourceLocation res)
     }
 
     /**
-     * Load a schematic into this world
+     * Load a structure into this world
      * and place it in the right position and rotation.
      *
      * @param worldObj  the world to load it in
-     * @param name      the schematics name
+     * @param name      the structures name
      * @param pos       coordinates
      * @param rotations number of times rotated
      */
-    public static void loadAndPlaceSchematicWithRotation(World worldObj, @NotNull String name, @NotNull BlockPos pos, int rotations)
+    public static void loadAndPlaceStructureWithRotation(World worldObj, @NotNull String name, @NotNull BlockPos pos, int rotations)
     {
         try
         {
-            @NotNull SchematicWrapper schematic = new SchematicWrapper(worldObj, name);
-            schematic.rotate(rotations);
-            schematic.placeSchematic(pos);
+            @NotNull StructureWrapper structureWrapper = new StructureWrapper(worldObj, name);
+            structureWrapper.rotate(rotations);
+            structureWrapper.placeStructure(pos);
         }
         catch (IllegalStateException e)
         {
-            Log.getLogger().warn("Could not load schematic!", e);
-        }
-    }
-
-    /**
-     * Rotate this schematic.
-     *
-     * @param times how many times to rotate the schematic.
-     */
-    public void rotate(int times)
-    {
-        if (times % NUMBER_OF_ROTATIONS == REVERSE_ROTATION)
-        {
-            //reverse rotate
-            rotate(EnumFacing.DOWN);
-        }
-        else
-        {
-            for (int i = 0; i < times; i++)
-            {
-                //normal rotate
-                rotate(EnumFacing.UP);
-            }
+            Log.getLogger().warn("Could not load structure!", e);
         }
     }
 
     /**
-     * Place a schematic into the world.
+     * Place a structure into the world.
      *
      * @param pos coordinates
      */
-    private void placeSchematic(@NotNull BlockPos pos)
+    private void placeStructure(@NotNull BlockPos pos)
     {
         setLocalPosition(pos);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Local variable 'delayedBlocks' could be declared final [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=pmd%3ALocalVariableCouldBeFinal)